runtime/lua/vim/lsp/_snippet.lua

```diff
@@ -156,10 +156,10 @@ P.seq = function(...)
   return function(input, pos)
     local values = {}
     local new_pos = pos
-    for _, parser in ipairs(parsers) do
+    for i, parser in ipairs(parsers) do
       local result = parser(input, new_pos)
       if result.parsed then
-        table.insert(values, result.value)
+        values[i] = result.value
```

Comment:
Previously empty `opt` were just omitted in `seq` (`nil` as last item in list is ignored), now they're `nil`.
I don't _think_ this breaks anything, wanted to highlight it nonetheless since I'm not 100% certain.
Do you think this could cause any issues @hrsh7th?

Reply:
I think this fix is make sense. Thank you! 👍🏼